### ISSUE TITLE
privacy problem with https://api.istex.fr/properties call ?

### ISSUE THREAD

**Issue by kerphi** [istex/istex-web-extension#3]
This part of the code could have problem with privacy because it requests the ISTEX API with an AJAX request (embedding the referer URL) when the browser starts (i thing at the first visited page but i'm wrong maybe?) :
https://github.com/istex/istex-web-extension/blob/57d27813d87bed7156ce725f6bc295eb6c2a9bd7/src/content_scripts/main.js#L483-L504

So we maybe have a privacy problem cause we capture "Referer" in the ISTEX logs coming from the first visited page. Here are example of line of logs (seen when working on the ISTEX's bibliomap) not related to any detected DOI or PMID or PII:

```
91.124.103.198 - - - [02/Mar/2017:22:33:38 +0000] "GET /properties HTTP/1.1" 200 333 "http://www.courrierinternational.com/article/2000/10/12/des-bottes-de-sept-lieues-a-moteur-diesel" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36" -
91.124.103.198 - - - [02/Mar/2017:22:22:36 +0000] "GET /properties HTTP/1.1" 200 333 "https://webmail.inist.fr/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36" -
89.15.33.55 - - - [02/Mar/2017:21:44:05 +0000] "GET /properties HTTP/1.1" 200 333 "http://www.yvelines.gouv.fr/Demarches-administratives/Cartes-grises-Permis-de-conduire-SIV-professionnel/Permis-de-conduire/Visite-medicale-suspension-et-annulation-du-permis-de-conduire/Annulation-de-permis-de-conduire" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.98 Safari/537.36 Vivaldi/1.6.689.40" -
```

Maybe there is a way to do not send the "Referer" when doing HTTP request fro a web extension? Or maybe there is a way to request the "/properties" URL only when needed (DOI or something  openurl'able in the page) ?
Or other ideas ?

**Comment by jj618** [comment]
You've got a real point here, prb fixed in [https://github.com/istex/istex-web-extension/commit/0faf61206e7af1120ac778311bff643b37ac47d2](url)

**Comment by niederlec** [comment]
Fixed in 1.2.5 version (published today July 11. 2017 on addons.istex.fr)
